zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```
@@ -2987,6 +3036,41 @@ components:
           additionalProperties: true
           description: All the variables visible in the root scope.
           type: object
+    MigrateProcessInstanceRequest:
+      type: object
+      properties:
+        targetProcessDefinitionKey:
+          description: The key of process definition to migrate the process instance to.
+          type: integer
+          format: int64
+        mappingInstructions:
+          type: array
+          items:
+            $ref: "#/components/schemas/MigrateProcessInstanceMappingInstruction"
+        operationReference:
+          description: >
+            A reference key chosen by the user that will be part of all records resulting from this operation.
+            Must be > 0 if provided.
+          type: integer
+          format: int64
+          minimum: 1
+      required:
+        - targetProcessDefinitionKey
+        - mappingInstructions
+    MigrateProcessInstanceMappingInstruction:
+      type: object
+      description: |
+        The mapping instructions describe how to map elements from the source process definition to the target process definition.
+      properties:
+        sourceElementId:
+          description: The element id to migrate from.
+          type: string
+        targetElementId:
+          description: The element id to migrate into.
```

```suggestion
          description: The element ID to migrate into.
```

🔧 Same as previous comment.